### PR TITLE
BookCard Component

### PR DESCRIPTION
**Pull request by lesliefernandez10**

### CODE REVIEW

**Review by josezumaque — CHANGES_REQUESTED** [review]

File: src/components/BookCard.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      type: Object,
+      required: true,
+    },
+  },
+};
+</script>
+
+<style scoped>
+.book {
+  display: grid;
```
[the remark this answers is not in the document]
Comment (lesliefernandez10) [reply src/components/BookCard.vue]: Done, thanks!

File: src/components/BookCard.vue [hunk 2]
```diff
@@ -72,15 +70,16 @@ export default {
   letter-spacing: 0.08em;
   text-transform: uppercase;
 }
-
+/*This class shortens any long titles or author names to two lines*/
 .line-clamp {
```
Comment (josezumaque) [inline src/components/BookCard.vue:74]: Since this is a utility class, should be moved into a global css file.

**Review by josezumaque — CHANGES_REQUESTED** [review]

File: src/components/BookCard.vue (hunk 2 above)
Comment (lesliefernandez10) [reply src/components/BookCard.vue]: Done!

**Review by josezumaque — APPROVED** [review]
Good job!

**Review by josezumaque — COMMENTED** [review]
Good job!

**Pull request merged**